tests/unit_tests/tests.js

```diff
@@ -53,4 +60,37 @@ test("Define.js: Exception on defining already defined", function(assert) {
 		new Error("Module someLib/existingModule already defined!"), // Expected exception
 		"Passed!");
 
-});
\ No newline at end of file
+});
+
+test("Content models: FilePath", function(assert) {
+
+	var FilePath = require('content/app/models/file-path');
+
+	// Standard scenario
+	var standardCase = new FilePath("a/b/c/someFile.bin");
+	assert.ok(standardCase.fileName === "someFile", "Standard scenario: file name");
+	assert.ok(standardCase.fileExtension === "bin", "Standard scenario: file name");
+	assert.ok(standardCase.fileNameWithExt === "someFile.bin", "Standard scenario: name + extension");
+	assert.ok(standardCase.fullPath === "a/b/c/someFile.bin", "Standard scenario: full path");
+
+	var pathSegments = standardCase.pathSegments;
+	var expectedSegments = ['a', 'b', 'c', 'someFile.bin'];
+	var areSame = true;
+	for (var i = 0; i < pathSegments.length; i++) {
+		if (pathSegments[i] !== expectedSegments[i]) {
+			areSame = false;
+			break;
+		}
+	}
+	assert.ok(areSame, "Standard scenario: path segments");
```

Comment:
:+1: 
There is deepEqual method which does this thing. I still do not know how I have not seen it.